### DISCUSSION TITLE
旋律展示的可视化，类似于pretty midi 可看钢琴卷帘窗或柱状和弦

### DISCUSSION THREAD

**Discussion by ChuxiJ** [Rainbow-Dreamer/musicpy#42] [Ideas]
旋律展示的可视化，类似于pretty midi 可看钢琴卷帘窗或柱状和弦

**Comment by Rainbow-Dreamer** [comment]
你好，旋律展示可视化这个功能已经在我为musicpy写的IDE [musicpy editor](https://github.com/Rainbow-Dreamer/musicpy_editor) 里面实现了，可以将任何乐理类型使用钢琴瀑布流演示，使用的是我写的一个智能钢琴软件的内核 [Ideal Piano](https://github.com/Rainbow-Dreamer/Ideal-Piano), 具体的演示效果可以看[这个视频](https://www.bilibili.com/video/BV1B5411N7tt/)
